mods/forestry-tools/clinometer.lua

```diff
@@ -101,86 +106,87 @@ local function update_hud(player, hud, hudname, text)
 end
 
 local function update_clinometer(player)
-	bg_angle = "0"
+	local pname = player:get_player_name()
+	instances[pname].bg_angle = "0"
```

Comment:
Given the discussion at our last meeting about Lua leaking memory with strings, I wonder if it would be best to cache this (and others) as an integer and then cast it to a string at the moment it is needed? Just thinking that this particular tool will be constantly cycling through strings as the player changes their viewing perspective.